### PR TITLE
#69 03 Misc/01 Linting issue reviewed

### PR DESCRIPTION
**Pull request by mlopezperez**
- README.md reviewed and corrected.
- All steps run and checked.

### CODE REVIEW

File: 03 Misc/01 Linting/README.md [hunk 1]
```diff
@@ -58,9 +60,9 @@ We are going to create a file `.eslintrc.json` (there are many [file formats opt
 {
   ...
   "scripts": {
--   "start": "webpack-dev-server"
-+   "start": "webpack-dev-server",
```
Comment (nasdan) [inline 03 Misc/01 Linting/README.md:62]: These lines are to note the reader: "hey, don't forget the comma", what do you think if we keep it?

File: 03 Misc/01 Linting/README.md [hunk 2]
```diff
@@ -252,18 +258,18 @@ const messageToDisplay = `average score ${averageScore}`;
     "node": true,
     "jquery": true
   },
-- "parser": "babel-eslint"
-+ "parser": "babel-eslint",
```
Comment (nasdan) [inline 03 Misc/01 Linting/README.md]: Same as above

File: 03 Misc/01 Linting/README.md [hunk 3]
```diff
@@ -278,16 +284,19 @@ const messageToDisplay = `average score ${averageScore}`;
   },
   "parser": "babel-eslint",
   "rules": {
--   "no-console": 0
-+   "no-console": 0,
```
Comment (nasdan) [inline 03 Misc/01 Linting/README.md]: Same as above

**Comment by nasdan** [comment]
Great stuff! I only commented you about keep code with `comma diff`

File: 03 Misc/01 Linting/README.md (hunk 1 above)
Comment (mlopezperez) [reply 03 Misc/01 Linting/README.md:62]: Fixed!

**Comment by mlopezperez** [comment]
Sorry for the late answer... I had a busy Monday! 
I've reviewed the code and kept the lines with/without commas where necessary.

**Comment by nasdan** [comment]
Don't worry, thank you for the changes. Nice work

**Pull request merged**